src/Aquifer.Data/Entities/BibleEntity.cs

```diff
@@ -20,6 +20,7 @@ public class BibleEntity : IHasUpdatedTimestamp
     [SqlDefaultValue("getutcdate()")]
     public DateTime Created { get; set; } = DateTime.UtcNow;
 
+    public LanguageEntity Language { get; set; } = null!;
```

Comment:
This was helpful for some content-loader scripting and as a nice byproduct adds a foreign key relationship there.